src/java.desktop/macosx/native/libawt_lwawt/awt/AWTEvent.m

```diff
@@ -309,6 +309,7 @@
     {0, 0, 0, 0, 0, 0}
 };
 
+static BOOL leftAltKeyPressed;
```

Comment:
I think all the occurrences of `leftAltKeyPressed` in AWTEvent.m can be removed.